src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/OnlineFormDocumentService.java

```diff
@@ -72,4 +100,47 @@ private void deleteOldMiniFormA(Map<String, Object> documentData, String authori
             }
         });
     }
-}
+
+    public CaseDocument generateConsentedInContestedMiniFormA(CaseDetails caseDetails, String authorisationToken) {
+
+        log.info("Generating 'Consented in Contested' Mini Form A for Case ID : {}", caseDetails.getId());
+
+        CaseDetails caseDetailsCopy = documentHelper.deepCopy(caseDetails, CaseDetails.class);
+
+        //translation is required to map the natureOfApplication2 values from contested to the consented equivalent
```

Comment:
not sure this comment is useful

Reply:
Its not abundantly clear that the field mappings for these 2 fields are the reason we require the translation. I figured best to document it somewhere.. Happy to remove though